plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -279,6 +279,8 @@
     public static final String ADD_COLUMN_OPERATION = "ADD COLUMNS";
     public static final String INSERT_OPERATION = "WRITE";
     public static final String MERGE_OPERATION = "MERGE";
+    public static final String UPDATE_OPERATION = "UPDATE"; // used by old Trino versions and Spark
```

Comment:
no action needed: `UPDATE` and `DELETE` should be actually recorded in the commit history by Trino https://github.com/trinodb/trino/issues/15763

Reply:
but spark may write UPDATE/DELETE there

Reply:
with the previous comment i wanted to emphasize that what the "old Trino" was doing was correct, while what the "current Trino" is doing is subject of improvement. :)